locales/templates/app.pot

```diff
@@ -541,3 +541,32 @@ msgctxt "dialog_clear_cookies_title"
 msgid "Clear Cookies?"
 msgstr ""
 
+#. Request desktop site strings
+#. Content description for Request desktop version button
+msgctxt "request_desktop_site_a11y"
+msgid "Request desktop version of this site"
+msgstr ""
+
+#. Notification displayed when desktop version is requested
```

Comment:
I'm wondering if the comment here is clear enough. Because "requested desktop version" can be understood either as "desktop version has been requested" (verb) or "this is the requested desktop version" (adjective) - which makes it hard to translate. Could we get a bit more clarification here please?